searx/plugins/self_info.py

```diff
@@ -46,14 +46,14 @@ def get_ip(request):
 
     # Initiate a GET request and set the outgoing proxies, if any were set in
     # settings.yml.
-    ip_info = get('http://ip-api.com/json/' + ip, proxies=outgoing_proxies).json()
+    ip_info = get('http://ipinfo.io/' + (ip + '/' if ip else '') + 'json/', proxies=outgoing_proxies).json()
```

Comment:
Haha, yes it is. I'll take care of it. I was mostly testing and wanted to see why the assertion test is failing on travis ci.